src/components/common/Item.js

```diff
@@ -205,6 +205,13 @@ const Item = ({ id, isChildren, showPinnedOnly }) => {
       );
     }
 
+    case ITEM_TYPES.SHORTCUT: {
+      if (item.extra?.shortcut?.target) {
+        return <Item isChildren id={item.extra?.shortcut?.target} />;
```

Comment:
Do we need to also expand any other props? (e.g. `showPinnedOnly`). If this is desired, it may make sense to copy all leftover props (so that even if we add any to the props signature in the future, they are still passed to it). For instance:
```ts
const Item = ({ id, isChildren, showPinnedOnly } = props) => {
    ...
    ...
    case ITEM_TYPES.SHORTCUT: {
      if (item.extra?.shortcut?.target) {
        return <{...props} id={item.extra?.shortcut?.target} />;
      }
      return null;
    }
    ...
```
Let me know if this is applicable or not 👍 otherwise I think it's a nice use of a recursive definition ✨